modules/renter/streamcache.go

```diff
@@ -172,9 +172,9 @@ func (sc *streamCache) SetStreamingCacheSize(cacheSize uint64) {
 func newStreamCache() *streamCache {
 	streamHeap := make(streamHeap, 0, defaultStreamCacheSize)
 	heap.Init(&streamHeap)
+	// cacheSize not set as it is persisted
 	return &streamCache{
 		streamMap:  make(map[string]*chunkData),
 		streamHeap: streamHeap,
-		cacheSize:  defaultStreamCacheSize,
```

Comment:
shouldn't this be set to `r.persist.StreamCacheSize`?

Reply:
oh, you will probably need to make `newStreamCache` a function of the renter. That should fix up all of the tests that broke as well.

Reply:
@DavidVorick i think you are looking at an out commit, can you check my next commit and let me know if your comment still applies

Reply:
The comment `cacheSize not set as it is persisted` can be removed now since the method was updated to take the cacheSize as an argument and set it

Reply:
ok